namesys/resolve/pathresolver_test.go

```diff
@@ -16,17 +16,17 @@ func TestResolveNoComponents(t *testing.T) {
 	}
 
 	_, err = resolve.Resolve(n.Context(), n.Namesys, n.Resolver, path.Path("/ipns/"))
-	if err != path.ErrNoComponents {
-		t.Fatal("Should error with no components (/ipns/).", err)
+	if err == nil {
```

Comment:
Probably not a big deal, but I'm curious, can we expect a specific error here and ultimately fail if we don't find the specific errors we're looking for?

Reply:
It appears @magik6k and I reviewed at the same time. We are leaving the same feedback here and so I'm going to resolve this particular conversation so it's not in two threads.